contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -495,6 +495,7 @@ func (s *SchedulerServer) prepareExecutorInfo(hks hyperkube.Interface) (*mesos.E
 	// running executors in a cluster.
 	execInfo.ExecutorId = executorinfo.NewID(execInfo)
 	execInfo.Data = data
+	log.V(1).Infof("started with executor id %v", execInfo.ExecutorId.GetValue())
```

Comment:
I pointed this out because it doesn't seem very useful to have the initial
executorID logged at V(1) when you're not going to see any debug warnings
because they're all at V(2). Suggestion: make them all V(1) or V(2).

On Tue, Feb 23, 2016 at 3:43 PM, Hai Huang <notifications@github.com> wrote:

> In contrib/mesos/pkg/scheduler/service/service.go
> <https://github.com/kubernetes/kubernetes/pull/21787#discussion_r53846499>
> :
>
> > @@ -495,6 +495,7 @@ func (s *SchedulerServer) prepareExecutorInfo(hks hyperkube.Interface) (*mesos.E
> >  	// running executors in a cluster.
> >  	execInfo.ExecutorId = executorinfo.NewID(execInfo)
> >  	execInfo.Data = data
> > +	log.V(1).Infof("started with executor id %v", execInfo.ExecutorId.GetValue())
>
> I was using V(2) for printing out error messages during problem scenarios,
> and V(1) for printing out more verbose messages during normal scenarios. I
> can make the change to either one if it is more conforming to the current
> coding style.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/21787/files#r53846499>.
>